src/vs/base/common/network.ts

```diff
@@ -220,11 +243,16 @@ class FileAccessImpl {
 	 * Returns the `file` URI to use in contexts where node.js
 	 * is responsible for loading.
 	 */
-	asFileUri(uri: URI): URI;
-	asFileUri(moduleId: string, moduleIdToUrl: { toUrl(moduleId: string): string }): URI;
-	asFileUri(uriOrModule: URI | string, moduleIdToUrl?: { toUrl(moduleId: string): string }): URI {
-		const uri = this.toUri(uriOrModule, moduleIdToUrl);
+	asFileUri(resourcePath: AppResourcePath | ''): URI {
+		const uri = this.toUri(resourcePath, require);
```

Comment:
Isn't this causing a layer break because we now depend on `require` as a global?

Reply:
Hm, maybe not because we previously passed in the global `require` from everywhere.

Reply:
to clarify, this was and still is the "local" `require`, not the global one

Reply:
Yeah its just TS language service telling me its the NodeJS one.